### PR TITLE
Add documents for CCXT, Bitbank, and Coincheck

### PR DESCRIPTION
**Pull request by macanudo527**
I just need to add documentation for the Pionex plugin in the `configuration_file.md`. I seemed to overlook that.

### CODE REVIEW

**Comment by macanudo527** [comment]
The failed tests are caused by links to unmerged files in PR #80

File: docs/configuration_file.md [hunk 1]
```diff
@@ -18,9 +18,11 @@
 * **[Introduction](#introduction)**
 * **[Data Loader Plugin Sections](#data-loader-plugin-sections)**
```
Comment (eprbell) [inline docs/configuration_file.md:19]: Add Coincheck and Pionex to the table of contents.

**Review by eprbell — CHANGES_REQUESTED** [review]
Minor changes needed.

**Comment by macanudo527** [comment]
This should be wrapped up.

File: docs/configuration_file.md [hunk 2]
```diff
@@ -84,12 +88,39 @@ username = <em>&lt;username&gt;</em>
...
 * Due to this discrepency, there may be duplicate locked savings/staked and flexible savings dividends around May 8th, 2021. Please review your payments around this time before processing your transactions with RP2.
-* [Currently only dust 'dribblets' of 100 or less crypto assets can be retrieved at once](https://dev.binance.vision/t/pagination-for-dustlog-asset-dividend-record-swap-history-bswap/4963). If you dust more than 100 crypto assets at one time the REST API will not be able to process the transactions successfully. 
+* [Currently only dust 'dribblets' of 100 or less crypto assets can be retrieved at once](https://dev.binance.vision/t/pagination-for-dustlog-asset-dividend-record-swap-history-bswap/4963). If you dust more than 100 crypto assets at one time the REST API will not be able to process the transactions successfully.
+<<<<<<< Updated upstream
```
Comment (eprbell) [inline docs/configuration_file.md:96]: The section marked by `<<<<<<<`... `>>>>>>>` is a Git conflict and needs to be resolved manually.

**Review by eprbell — CHANGES_REQUESTED** [review]
A conflict needs to be resolved.

**Comment by macanudo527** [comment]
Fixed the conflict.

**Review by eprbell — APPROVED** [review]
LGTM

**Pull request merged**